demos/toolbar/default-toolbar.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
A few comments on the demo pages:

- It may be useful to show the other toolbar sections being used. For example, a common pattern is to have a site title in the start section, and an array of icons in the end section, a la Inbox. Perhaps we should demonstrate this?
- It may be good to show how a toolbar looks within an RTL context (e.g. with an ancestor element that has `dir="rtl"` attached to it).
- I feel like we only need one demo page to show both normal and fixed toolbars. We simply need to wrap each toolbar and corresponding section in a scrollable container and put it on the page. For example:

```html
<style>
  .demo-container {
    max-height: 500px;
    overflow: scroll;
  }
</style>

 <!-- ... -->
  <h2>Normal Toolbar</h2>
  <div class="demo-container">
    <div class="mdc-toolbar ..."></div>
    <p>Lorem ipsum .....</p>
  </div>

  <h2>Fixed Toolbar</h2>
  <div class="demo-container">
    <div class="mdc-toolbar mdc-toolbar--fixed ..."></div>
    <p class="mdc-toolbar-fixed-adjust">Lorem ipsum...</p>
  </div>
```

By putting all toolbars on one page, we not only make it easier for users to see the different types of toolbars available all in one place (instead of two separate pages), but we explicitly show how toolbars can be used within subsections of a page, and not just as a singular element on a page, which is a limitation with other pre-existing Material Components implementations.